File: tmp-mei/fgen4255027576711118260/base/flows/TerminalOnInfiniteError_0.java
package base.flows;
public interface TerminalOnInfiniteError_0{
TerminalOnInfiniteError_0 $self = new TerminalOnInfiniteError_0Impl();
base.Info_0 $hash$imm();
static base.Info_0 $hash$imm$fun(base.flows.TerminalOnInfiniteError_0 $this) {
  return ((base.Info_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Info_0) null;
}})
);
}
}

File: tmp-mei/fgenjs5030319945734616766/base/flows/TerminalOnInfiniteError_0.js
import * as rt from "../rt/main.js";
export class base$$flows$$TerminalOnInfiniteError_0 {
  static $self = new base$$flows$$TerminalOnInfiniteError_0();

  $hash$imm() {
  return base$$flows$$TerminalOnInfiniteError_0.$hash$imm$fun(this);
}

  static $hash$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}
